### DISCUSSION TITLE
blog/2021/05/13/lpc-in-python

### DISCUSSION THREAD

**Discussion by giscus** [kunigami/kunigami.github.io#9] [Announcements]
# blog/2021/05/13/lpc-in-python

NP-Incompleteness: 

https://www.kuniga.me/blog/2021/05/13/lpc-in-python.html

**Comment by qhanson** [comment]
Hi, I listen the generated audio. It is a litte noised. Is it normal? The reason is two folds.
First, we assume the audio is follows the liner prediction.
Second, we ignore the bias in the Xa=b euqation.
